cicecore/drivers/nuopc/cmeps/ice_prescribed_mod.F90

```diff
@@ -7,33 +7,15 @@ module ice_prescribed_mod
   ! Ice/ocean fluxes are set to zero, and ice dynamics are not calculated.
   ! Regridding and data cycling capabilities are included.
 
-  use ESMF, only : ESMF_Clock, ESMF_Mesh, ESMF_SUCCESS, ESMF_FAILURE
+  use ESMF, only : ESMF_GridComp, ESMF_Clock, ESMF_Mesh, ESMF_SUCCESS, ESMF_FAILURE
   use ESMF, only : ESMF_LogFoundError, ESMF_LOGERR_PASSTHRU, ESMF_Finalize, ESMF_END_ABORT
 
-#ifndef CESMCOUPLED
```

Comment:
I think we should leave this section, and add a UFSCOUPLED or a NO_CDEPS FLAG .

Reply:
It seems cleaner to admit the dependency on CDEPS. Is this ok @anton-seaice?

Reply:
Consensus seems to be to go with the PR as currently written